<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>useActionWindow</class>
 <widget class="QMainWindow" name="useActionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Windows Manager - Autoplay Groups</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(58, 58, 58);
color:rgb(227, 227, 227)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>460</y>
      <width>651</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Sans</family>
     </font>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Sans</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Delete Group</string>
     </property>
    </widget>
    <widget class="QComboBox" name="deleteBox">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>341</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Sans</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(52, 52, 52);
background-color:rgb(144, 144, 144)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteBtn">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>20</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Sans</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(40, 40, 40)</string>
     </property>
     <property name="text">
      <string>âœ“</string>
     </property>
    </widget>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>651</width>
      <height>311</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Sans</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(91, 91, 91)</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>649</width>
       <height>309</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="stopBtn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>370</y>
      <width>551</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Sans</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(52, 52, 52);
background-color:rgb(59, 218, 218)</string>
    </property>
    <property name="text">
     <string>No Group Playing</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="createAction">
   <property name="text">
    <string>Create New Instance</string>
   </property>
  </action>
  <action name="openAction">
   <property name="text">
    <string>Open Existing Instances</string>
   </property>
  </action>
  <action name="quitAction">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="configAction">
   <property name="text">
    <string>Config Menu</string>
   </property>
  </action>
  <action name="locationAction">
   <property name="text">
    <string>Saved .exe Locations</string>
   </property>
  </action>
  <action name="autoplayAction">
   <property name="text">
    <string>Autoplay Speed</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
